platforms/posix/src/main.cpp

```diff
@@ -578,26 +578,30 @@ void print_usage()
 bool is_already_running(int instance)
 {
 	const std::string file_lock_path = std::string(LOCK_FILE_PATH) + '-' + std::to_string(instance);
-	struct flock fl;
 	int fd = open(file_lock_path.c_str(), O_RDWR | O_CREAT, 0666);
 
 	if (fd < 0) {
+		close(fd);
 		return false;
 	}
 
+	struct flock fl {};
+
```

Comment:
The git format/pretty hook is forcing these whitespace additions, I would prefer to leave them out.

Reply:
So do I :/

Reply:
I put the intialized `struct flock fl {}` call back on it's original line because the added whitespace was worse than having the declaration separated from its' usage. :}